### PR TITLE
Net 4230/add tcp to basic acceptance test

### PR DESCRIPTION
**Pull request by missylbytes**
Changes proposed in this PR:
Adds tcp to basic acceptance tests

How I've tested this PR:
Run the tests
How I expect reviewers to test this PR:
Check out that the acceptance tests pass!

Checklist:
- [ ] Tests added

### CODE REVIEW

File: acceptance/tests/api-gateway/api_gateway_test.go [hunk 1]
```diff
@@ -79,12 +81,15 @@ func TestAPIGateway_Basic(t *testing.T) {
 				k8s.RunKubectlAndGetOutputE(t, ctx.KubectlOptions(t), "delete", "-k", "../fixtures/bases/api-gateway")
 			})
 
-			logger.Log(t, "creating target server")
+			logger.Log(t, "creating target http server")
 			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.DebugDirectory, "../fixtures/cases/static-server-inject")
 
-			logger.Log(t, "patching route to target server")
+			logger.Log(t, "patching route to target http server")
 			k8s.RunKubectl(t, ctx.KubectlOptions(t), "patch", "httproute", "http-route", "-p", `{"spec":{"rules":[{"backendRefs":[{"name":"static-server","port":80}]}]}}`, "--type=merge")
```
Comment (missylbytes) [inline acceptance/tests/api-gateway/api_gateway_test.go:88]: Need to follow this pattern for tcp-route also

File: acceptance/tests/api-gateway/api_gateway_test.go [hunk 2]
```diff
@@ -199,8 +235,11 @@ func TestAPIGateway_Basic(t *testing.T) {
 
 			// Test that we can make a call to the api gateway
 			// via the static-client pod. It should route to the static-server pod.
-			logger.Log(t, "trying calls to api gateway")
+			logger.Log(t, "trying calls to api gateway http: ", targetAddress)
```
Comment (missylbytes) [inline acceptance/tests/api-gateway/api_gateway_test.go:238]: remove targetAddress in this logging call

**Comment by wilkermichael** [comment]
A reminder that the 1.2.x branch is currently under a code freeze. If this isn't a bug fix/test introduction, and unless it's a critical feature for the 1.2.x-rc can you please refrain from merging this until after the 1.2.0-rc1 release (Tuesday, June 13th)

File: acceptance/tests/api-gateway/api_gateway_test.go [hunk 3]
```diff
@@ -79,12 +81,15 @@ func TestAPIGateway_Basic(t *testing.T) {
...
-			logger.Log(t, "creating target server")
+			logger.Log(t, "creating target http server")
 			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.DebugDirectory, "../fixtures/cases/static-server-inject")
 
-			logger.Log(t, "patching route to target server")
+			logger.Log(t, "patching route to target http server")
 			k8s.RunKubectl(t, ctx.KubectlOptions(t), "patch", "httproute", "http-route", "-p", `{"spec":{"rules":[{"backendRefs":[{"name":"static-server","port":80}]}]}}`, "--type=merge")
 
+			logger.Log(t, "creating target tcp server")
+			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.DebugDirectory, "../fixtures/bases/static-server-tcp")
```
[the remark this answers is not in the document]
Comment (missylbytes) [reply acceptance/tests/api-gateway/api_gateway_test.go:96]: fixed

**Comment by andrewstucki** [comment]
Hey @missylbytes clarified with Michael that these sort of test introductions can indeed be merged in this frozen window, so I think we're good for marking this as ready for review once the acceptance tests pass.

File: acceptance/tests/api-gateway/api_gateway_test.go (hunk 2 above)
Comment (missylbytes) [reply acceptance/tests/api-gateway/api_gateway_test.go]: removed

**Review by andrewstucki — APPROVED** [review]

**Pull request merged**